analytics.js

```diff
@@ -772,6 +779,7 @@ if (process.browser) {
       let event = [...instance.toBeProcessedArray[i]];
       let method = event[0];
       event.shift();
+      logger.debug("=====from init, calling method:: ", method)
```

Comment:
Logger format need to be aligned as mentioned in earlier comments. it needs to be aligned